benefits/oauth/hooks.py

```diff
@@ -48,6 +48,7 @@ def post_logout(cls, request):
     )
     def claims_verified_eligible(cls, request, claims_request, claims_result):
         super().claims_verified_eligible(request, claims_request, claims_result)
+        session.update(request, logged_in=True)
```

Comment:
I suppose the new name is not as technically precise (the user has to make it through `authorize` to have this status set), but I think it makes more sense from a conceptual, high-level perspective?

Reply:
Agree with you, this is more clear